ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js

```diff
@@ -0,0 +1,362 @@
+          borderColor="#3182CE"
+          borderWidth={3}
+          size="lg"
+          disabled={isPublishDisabled()}
+          onClick={() => setIsEditorOpen(true)}
+        >
+          Add New
+        </Button>
+        <Button
+          colorScheme="blue"
+          variant={isEditorOpen ? 'outline' : 'solid'}
+          borderColor="#3182CE"
+          borderWidth={3}
+          size="lg"
+          onClick={() => setIsEditorOpen(false)}
+        >
+          Past Sermons
+        </Button>
+      </Stack>
+      {!isEditorOpen && (
+        <>
+          <List spacing="2" pt={3}>
+            {sermonNotesList.map((sermonNoteItem) => (
+              <ListItem key={sermonNoteItem.sermonId}>
+                <Box p="3" borderRadius="lg" borderWidth="1px">
+                  <Flex direction={['column', 'row']} spacing={1}>
+                    <Box maxW="13rem" pr={6}>
+                      <Image
+                        src={sermonNoteItem.imageLink}
+                        fallbackSrc="https://hongkong.sub.hmcc.net/wp-content/uploads/Screenshot-2020-09-04-at-6.39.50-PM.png"
+                      />
+                    </Box>
+                    <Stack direction="column" spacing={2}>
+                      <Heading size="md">{sermonNoteItem.title}</Heading>
+                      <Grid
+                        templateColumns={['repeat(1,2fr)', 'repeat(2,2fr)']}
+                        gap={1.5}
+                      >
+                        <HStack alignments="center" gap={1}>
+                          <FaRegCalendarAlt />
+                          <Text> Date: {sermonNoteItem.date} </Text>
+                        </HStack>
+                        <HStack alignments="center" gap={1}>
+                          <IoPeopleOutline />
+                          <Text>Speaker: {sermonNoteItem.speaker}</Text>
+                        </HStack>
+                        <HStack alignments="center" gap={1}>
+                          <MdOutlineVideoLibrary />
+                          <Text>
+                            Sermon Series: {sermonNoteItem.sermonSeries}
+                          </Text>
+                        </HStack>
+                        <HStack alignments="center" gap={1}>
+                          <IoBookOutline />
+                          <Text>Passage: {sermonNoteItem.passage}</Text>
+                        </HStack>
+                        <HStack alignments="center" gap={1}>
+                          <BiDonateHeart />
+                          <Text>
+                            Service Type: {sermonNoteItem.serviceType}
+                          </Text>
+                        </HStack>
+                      </Grid>
+                    </Stack>
+                    <Spacer />
+                    <Stack
+                      pt={[3, 3, 3, 0]}
+                      spacing={1}
+                      direction={['column', 'column', 'column', 'row']}
```

Comment:
it's not rly necessary for u to do all the breakpoints like this. Doing smth like 
`direction={['column', 'row']}` should be enough, as our breakpoints are base for mobile and md for desktop.

Reply:
okayy sure will change back to original because was told to do like this instead